backend/getDataFiber.js

```diff
@@ -134,7 +146,14 @@ function getDataFiber(fiber: Object, getOpaqueNode: (fiber: Object) => Object):
 }
 
 function setInProps(fiber, path: Array<string | number>, value: any) {
-  fiber.pendingProps = copyWithSet(fiber.memoizedProps, path, value);
+  const inst = fiber.stateNode;
+  fiber.pendingProps = copyWithSet(inst.props, path, value);
```

Comment:
@sebmarkbage @acdlite is this logic still applicable given the recent changes to how fiber will work with async?